Makefile

```
@@ -329,3 +330,10 @@ MOCKERY := $(shell which mockery)
 mockery:
 	# will look into .mockery.yaml for configuration
 	$(MOCKERY)
+
+.PHONY: cert-manager
+cert-manager: kustomize
+	helm repo add jetstack https://charts.jetstack.io
+	kubectl create namespace cert-manager --dry-run=client -o yaml | kubectl apply -f -
+	helm upgrade --namespace cert-manager --version $(CERT_MANAGER_VERSION) --install cert-manager jetstack/cert-manager --set installCRDs=true
```

Works both ways, just mentioning that the `helm upgrade` commands supports creating the namespace:
> --create-namespace                           if --install is set, create the release namespace if not present

```suggestion
	helm upgrade --namespace cert-manager --version $(CERT_MANAGER_VERSION) --install cert-manager jetstack/cert-manager --set installCRDs=true --create-namespace
```